### DISCUSSION TITLE
PyInstaller pack fail

### DISCUSSION THREAD

**Discussion by ztj1993** [fastapi/fastapi#7042] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Example

Here's a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) with my use case:

```Python
import uvicorn
from fastapi import FastAPI

app = FastAPI()


@app.get("/")
def read_root():
    return {"Hello": "World"}


if __name__ == '__main__':
    uvicorn.run(
        app,
        host='0.0.0.0',
        port=80,
    )
```

### Description

```
pyinstaller -F a.py
```
Packing cannot be completed.

### Environment

* OS: Windows
* FastAPI Version: 0.61.2

To know the FastAPI version use:

```bash
python -c "import fastapi; print(fastapi.__version__)"

0.61.2
```

* Python version:

To know the Python version use:

```bash
python --version

Python 3.8.5
```

### Additional context

package is the latest version.

**Comment by soumojit** [comment, marked as answer]
@ztj1993 can you please try the same with python3.7 instead of python3.8?

Please check,
-- https://github.com/pyinstaller/pyinstaller/issues/4311
-- https://github.com/pyinstaller/pyinstaller/issues/5281

**Comment by ztj1993** [comment]
@soumojit Thanks, python 3.7 is available after testing!

**Comment by tiangolo** [comment]
Thanks for the help here @soumojit ! :clap: :bow: 

Thanks for reporting back and closing the issue @ztj1993  :+1:
